source/includes/write/transaction.php

```diff
@@ -0,0 +1,54 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your connection URI');
+$client = new MongoDB\Client($uri);
+
+// begin-callback
+$receipts = $client->bank->receipts;
+$checking = $client->bank->checking_accounts;
+$saving = $client->bank->saving_accounts;
+
+$accountId = '5678';
+$transferAmount = 1000.00;
+
+$callback = function (MongoDB\Driver\Session $session) 
+    use ($checking, $saving, $receipts, $accountId, $transferAmount): void {
+
+    $checking->updateOne(
+           ['account_id' => $accountId],
+           ['$inc' => ['balance' => -$transferAmount]],
+           ['session' => $session]
+    );
+    
+    $saving->updateOne(
+        ['account_id' => $accountId],
+        ['$inc' => ['balance' => $transferAmount]],
+        ['session' => $session]
+    );
+
+    $summary = sprintf(
+        "SAVINGS +%u CHECKING -%u", $transferAmount, $transferAmount
+    );
+
+    $receipts->insertOne([
+        'account_id' => $accountId,
+        'summary' => $summary,
+        'timestamp' => new MongoDB\BSON\UTCDateTime(),
+    ],
+    ['session' => $session]);
```

Comment:
```suggestion
    $receipts->insertOne(
        [
            'account_id' => $accountId,
            'summary' => $summary,
            'timestamp' => new MongoDB\BSON\UTCDateTime(),
        ],
        ['session' => $session],
    );
```

Optional code formatting suggestion to avoid having anything _within_ the method aligned to the parent level.

Reply:
used a code formatter to fix these issues